### DISCUSSION TITLE
现在开不了JAVA类型的服务器了吗？

### DISCUSSION THREAD

**Discussion by No1Guapi** [natfrp/wiki#246] [水深火热]
RT，想要开个小服务器和朋友一起玩，但是没有找到JAVA类型，朋友连不上服务器，日志甚至都没有连接显示，不知道该怎么搞，有好哥哥知道吗？

**Comment by fakeboboliu** [comment]
我不太懂为什么你都找到文档的 repo 了还没看过文档

**Reply by No1Guapi** [reply]
抱歉，我对这个几乎就是小白一个。文档的repo是指报告之类的吗？我说的日志是SAKURAFRP客户端的运行日志，因为找不到JAVA类型的服务器，所以随便选择了一个服务器类型后顺利打通了隧道，但是给出地址后朋友们却无法连接到服务器，想了想应该是服务器类型的问题。只是我不管在哪都没有找到frp的JAVA类型服务器，看过论坛后大家好像没有类似的问题……（挠头）

**Reply by No1Guapi** [reply]
如果是网站的帮助文档的话，我也翻过了，但似乎没有这样的问题描述。一时找不到办法，所以就在论坛上询问一下

**Reply by No1Guapi** [reply]
抱歉，是我没看仔细……之前使用过frp的时候很顺利所以忽略了一些东西，以为只是服务器类型的问题，没有注意看配置文件。已经找到了，十分感谢
